packages/dds/sequence/src/test/tsconfig.json

```diff
@@ -1,18 +1,16 @@
 {
-	"extends": ["../../../../../common/build/build-common/tsconfig.test.json"],
+	"extends": "../../../../../common/build/build-common/tsconfig.test.node16.json",
+	"compilerOptions": {
+		"rootDir": "./",
+		"outDir": "../../lib/test",
+		"types": ["mocha", "node"],
+		"noImplicitAny": false,
+		"noUnusedLocals": false, // Need it so memory tests can declare local variables just for the sake of keeping things in memory,
```

Comment:
Pre-existing - better to have these handled explicitly where they are. Not suggesting change in this PR.

Reply:
AB#7423